### PR TITLE
Fixes #157, #156

### LINKED ISSUE #156: Show @ in profile page next to SlackId
We show @ on profiles list, but not on profile page

### LINKED ISSUE #157: Remove @ from slack id
Some people provide slack id with @ at the beginning, it should be stripped

### PR DESCRIPTION
**Pull request by jedrzej**
Fixes #157 Remove @ from slack ID
Fixes #156 Show @ next to slack ID in profile page

---

This change is [Reviewable](https://reviewable.io/reviews/x-team/x-map/174)

### CODE REVIEW

File: client/js/components/forms/ProfileForm.js [hunk 1]
```diff
@@ -43,8 +47,8 @@ class ProfileForm extends Component {
 
           <fieldset className="form-group">
             <label>Slack ID</label>
-            <input type="text" className="form-control" placeholder="Slack ID" value={this.state.slackId}
-              onChange={this.onInputChange.bind(this, 'slackId')} maxLength="64" required/>
+            <input type="text" className="form-control" placeholder="Slack ID2" value={this.state.slackId}
```
Comment (bernardodiasc) [inline client/js/components/forms/ProfileForm.js]: why `Slack ID2` in the palceholder?

**Comment by bernardodiasc** [comment]
looks good! just let a question about something that look a typo in the input placeholder.

**Comment by jedrzej** [comment]
Good catch, I'll fix and merge.

**Comment by jedrzej** [comment]
Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/x-team/x-map/174)_

**Comment by jedrzej** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/x-team/x-map/174)_

**Comment by jedrzej** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_[client/js/components/forms/ProfileForm.js, line 50 [r1]](https://reviewable.io:443/reviews/x-team/x-map/174#-KHC4hja0UKwdzvEexVY-r1-50:-KHL8QxJrBLFk_1yo8Ck:-499146009) ([raw file](https://github.com/x-team/x-map/blob/4100234b7c93b8e40d8beead88739510490e2cf5/client/js/components/forms/ProfileForm.js#L50)):_
<details><summary>

<i>Previously, bernardodiasc (Bernardo Dias) wrote…</i></summary>



> why `Slack ID2` in the palceholder?</details>
> Done.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/x-team/x-map/174)_

**Comment by jedrzej** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/x-team/x-map/174)_

**Comment by jedrzej** [comment]
Reviewed 1 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/x-team/x-map/174)_

**Comment by bernardodiasc** [comment]
Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---

_Comments from [Reviewable](https://reviewable.io:443/reviews/x-team/x-map/174)_

**Pull request merged**